pkg/windows/build_pkg.ps1

```diff
@@ -73,15 +108,17 @@ If (Test-Path "C:\Program Files (x86)") {
     Invoke-WebRequest -Uri 'http://repo.saltstack.com/windows/dependencies/32/vcredist_x86_2008_mfc.exe' -OutFile "$instDIR\vcredist.exe"
 }
 
-
 # Build Installer
 if ($version -eq $null) {
 	$gitDesc = git describe
 	if ($gitDesc -eq $null) {
-		$version = 'Beta'
+		$version = read-host "What would you like to name this build?"
 	} else {
 		$version = $gitDesc
-	}
+    }
+    else {
```

Comment:
the else statement isn't needed if it is just contains a continue.  removing it will do the same thing